torch/_C/__init__.pyi.in

```diff
@@ -17,16 +19,17 @@ T = TypeVar('T')
 
 # Defined in torch/csrc/Device.cpp
 class device:
-    type: str
-    index: _int
+    type: str  # THPDevice_type
+    index: _int  # THPDevice_index
 
+    # THPDevice_pynew
     @overload
-    def __init__(self, device: Union[_int, str]) -> None: ...
+    def __init__(self, device: Device) -> None: ...
```

Comment:
`Device` includes `None`, which works when `device` is a keyword in other functions, but here `torch.device(None)` actually gives a `TypeError`. So I think this change should be reverted.